### DISCUSSION TITLE
When installing from Windows, will Windows be forever disabled (my hope)?

### DISCUSSION THREAD

**Discussion by jerryhall** [sebanc/brunch#1042] [Q&A]
When I install ChromeOS via Brunch via the Windows method in the readme will this ultimately overwrite my windows installation? 

My goal is to rid my i3 notebook of Windows all together to recover space (HD is 500gb so not desperate) and forever be able to use the Chrome OS or access a Linux shell. 

Thank you!

**Comment by kerta1n** [comment]
Hello, there are multiple ways to install ChromeOS through Windows. if you use the [USB/SD method](https://github.com/sebanc/brunch#install-chromeos-on-a-usb-flash-drive--sd-card), you can boot into that external drive and follow [this](https://github.com/sebanc/brunch#install-chromeos-on-hdd-from-chromeos) to wipe the HDD of Windows and have only ChromeOS. However, it is not recommended to run ChromeOS on a HDD as no official Chromebooks do this and your ChromeOS might fail itself from loading too slow. This is the [dual-boot method](https://github.com/sebanc/brunch#dual-boot-chromeos-from-your-hdd-1), but I would recommend making a ChromeOS drive and buying an SSD, removing your existing HDD and [installing ChromeOS on your SSD](https://github.com/sebanc/brunch#dual-boot-chromeos-from-your-hdd-1). The SSD does not have to be that large as Chromebooks typically only come with 8-16GB of space, so a small 120GB SSD should be just fine if you aren't storing everything on that machine.

**Reply by jerryhall** [reply]
Ok excellent, thanks @kerta1n! I'm on the hunt for smaller SSDs, 100-200GB ought to do fine.

I'll work out a system to replicate this effort. Any feedback or contact from anyone anytime is welcome on this process. That's because I'm doing it to recycle old notebooks people give me, so I can put them them into the hands of current or recent-past un-housed San Diego residents. Many have cells but, it's tough when they need more power to help them get back into the workforce. I've only done one so far (2 yrs ago) and the woman thanks me every time she sees me (totally unnecessary but, she's grateful). So, I'm on my second now and looking to do more. It is a solid way to help. Thanks again!
